src/app/App.tsx

```diff
@@ -28,12 +29,20 @@ const switchClassname = style({
 });
 
 const App: React.ComponentType = () => {
+
     const intl = useIntl();
     const { rbac, server, isOrgAdmin } = useApp();
     const location = useLocation();
     const insights = getInsights();
     const [ usingExperimental, setUsingExperimental ] = React.useState<boolean>(false);
 
+    const url = window.location.pathname;
+    const filename = url.substring(url.lastIndexOf('/') + 1);
```

Comment:
This is somewhat brittle, but can work for now - The problem we have is that there is no way to know if this result is actually a bundle  (because we have no access to notifications) - We can follow up on this if we see some issues.
Could you update instead of filename call it `bundle` ?

Reply:
Yeah, i understand that. I can start working towards what you mentioned in slack... because I also need to add the bundle displayName to the page header, so as you said, it would make more sense to split it since we have access to the bundle there.

Reply:
Something else - If the `serviceName` is `integrations`, the `bundle` won't make sense. I think we can set it to an empty string and default to `/user-preferences/notifications/` in the userPreferences.

We could probably have a different message in there, but its fine for now.